idd/Energy+.idd.in

```diff
@@ -31865,6 +31865,7 @@ ZoneHVAC:FourPipeFanCoil,
         \key CyclingFan
         \key VariableFanVariableFlow
         \key VariableFanConstantFlow
+        \key MultiStageFan
```

Comment:
@mjwitte @Myoldmopar I contacted Jim Pegues of Carrier, he prefers the name "MultiSpeedFan" over "MultiStageFan".  So, the key name may have to change.